packages/ListBox/test/helpers/selectors.js

```diff
@@ -6,7 +6,7 @@ const selectors = {
   noResults: "[data-pka-anchor='no-results']",
   popover: "[data-pka-anchor='popover.content']",
   popoverList: "[data-pka-anchor='styled-list']",
-  trigger: "[data-pka-anchor='listbox-trigger']",
+  trigger: "[data-pka-anchor='list-box-trigger']",
```

Comment:
You can find this in ListBoxBrowser.styles.js, I guess you will need to adjust this one once this is release?
```js
    [data-pka-anchor="listbox-trigger"] {
      border: 0;
      display: none;
    }
```